collectors/template.yaml

```diff
@@ -0,0 +1,88 @@
+name: Collector name
+title: TBD Monitoring
+virtual_integrations:
+  list:
```

Comment:
what is this?

Reply:
This is for integrations we want to expose which do not have a dedicated collector.
Eg: Ubuntu, Redhat, Other Linux Distros will create a page which is mostly a copy of the OS Metrics (or some page) and only the description changes..
Eg 2: AWS RDS PostgreSQL and other SaaS Postgres offerings